composer.json

```diff
@@ -8,7 +8,8 @@
     }
   ],
   "require": {
-    "phpunit/phpunit": "^5.5|^6"
+    "php": ">=5.6",
+    "phpunit/phpunit": "^5.5|^6.5"
```

Comment:
Why do we need this?

Reply:
It seems that the ```PHPUnit``` require ```5.5+``` or ```6.5+```.
So I add the required PHP version ```>=5.6``` to check whether the environment is the correct PHP version.

Reply:
The environment check is fine, but why does the `^6` constraint need changing to `^6.5`?

Reply:
Because I want to let ``PHPUnit``` be the latest version.

Reply:
That doesn't make sense. `^6` already permits the latest version; `^6.5` just prevents earlier versions, which is not desirable unless you can prove it's necessary to fix this issue.

Reply:
All right, I will add the ```PHPUnit``` version back.